app/src/main/res/navigation/nav_graph.xml

```diff
@@ -184,6 +188,10 @@
         <action
             android:id="@+id/action_browserFragment_to_homeFragment"
             app:destination="@id/homeFragment" />
+        <action
```

Comment:
Maybe a bit more suggestive name like action_browserFragment_to_homeFragment_popUp?

Also, is the old action still needed? If not, maybe we could remove it.